src/GridView/CardGrid.js

```diff
@@ -0,0 +1,32 @@
+import React, { Component } from 'react';
+import { Card, CardTitle, CardText, CardActions } from 'material-ui/Card';
+import FlatButton from 'material-ui/FlatButton';
+import './CardGrid.css';
+
+class CardGrid extends Component {
+	render() {
+		return (
+			<div className="card-grid d-flex">
+				{this.props.entities.map((entity) => {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Remove the unused function parameter "entity". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://bpt-lab.org/sonarqube/coding_rules#rule_key=javascript%3AUnusedFunctionArgument)